pom.xml

```diff
@@ -2970,7 +2970,6 @@
               <spark.ui.showConsoleProgress>false</spark.ui.showConsoleProgress>
               <spark.unsafe.exceptionOnMemoryLeak>true</spark.unsafe.exceptionOnMemoryLeak>
               <spark.memory.debugFill>true</spark.memory.debugFill>
-              <spark.hadoop.hadoop.security.key.provider.path>test:///</spark.hadoop.hadoop.security.key.provider.path>
```

Comment:
To be safe, shall we keep this, @LuciferYang ? We have `spark.ui.enabled` in both places.

Reply:
OK ~